source/components/JsxParser.test.js

```diff
@@ -71,26 +71,99 @@ describe('JsxParser Component', () => {
         <JsxParser
           bindings={{ foo: 1 }}
           jsx={`
-            <div className={foo === 1 ? 'yes' : 'no'}>
-              {foo !== 1 ? 'bar' : 'foo'}
+            <div className={foo === 1 ? 'isOne' : 'isNotOne'}>
+              {foo !== 1 ? 'isNotOne' : 'isOne'}
             </div>
           `}
         />
       )
 
-      expect(rendered.childNodes[0].classList).toContain('yes')
-      expect(rendered.childNodes[0].textContent.trim()).toEqual('bar')
+      expect(rendered.childNodes[0].classList).toContain('isOne')
+      expect(rendered.childNodes[0].textContent.trim()).toEqual('isOne')
     })
 
     it('should handle test predicate returned value ', () => {
       const { rendered } = render(
         <JsxParser
-          jsx={'<p>{true && false ? "a" : "b"}</p><p>{false || true ? "a" : "b"}</p>'}
+          jsx={
+            '<p>{true && true ? "a" : "b"}</p>' +
+            '<p>{true && false ? "a" : "b"}</p>' +
+            '<p>{true || false ? "a" : "b"}</p>' +
+            '<p>{false || false ? "a" : "b"}</p>'
+          }
+        />
+      )
+
+      expect(rendered.childNodes[0].textContent).toEqual('a')
+      expect(rendered.childNodes[1].textContent).toEqual('b')
+      expect(rendered.childNodes[2].textContent).toEqual('a')
+      expect(rendered.childNodes[3].textContent).toEqual('b')
+    })
+  })
+
+  describe('conditional || rendering', () => {
+    it('should handle boolean test value ', () => {
+      const { component, rendered } = render(<JsxParser jsx={
+      '<p falsyProp={false || "fallback"} truthyProp={true || "fallback"}>'
+        + '(display "good": {"good" || "fallback"}); (display "fallback": {"" || "fallback"})'
+        + '</p>'
+      }
+      />)
+
+      expect(rendered.childNodes[0].textContent)
+        .toEqual('(display "good": good); (display "fallback": fallback)')
+
+      expect(component.ParsedChildren[0].props.falsyProp).toBe("fallback")
+      expect(component.ParsedChildren[0].props.truthyProp).toBe(true)
+    })
+
+    it('should handle evaluative', () => {
+      const { component, rendered } = render(
+        <JsxParser
+          bindings={{ foo: 1 }}
+          jsx={`
+            <div truthyProp={foo === 1 || 'fallback'} falseyProp={foo !== 1 || 'fallback'}>
+              {foo === 1 || 'trueFallback'}{foo !== 1 || 'falseFallback'}
+            </div>
+          `}
         />
       )
+      expect(component.ParsedChildren[0].props.truthyProp).toBe(true)
+      expect(component.ParsedChildren[0].props.falseyProp).toBe("fallback")
+      expect(rendered.childNodes[0].textContent.trim()).toEqual('falseFallback')
+    })
+  })
+
+  describe('conditional && rendering', () => {
```

Comment:
I didn't really need the same tests for && but I added them to keep it balanced...